thinc/api.py

```diff
@@ -0,0 +1,83 @@
+# flake8: noqa
+from .loss import categorical_crossentropy, L1_distance, cosine_distance
+from .config import Config
+from .initializers import normal_init, uniform_init, xavier_uniform_init, zero_init
+from .loss import categorical_crossentropy, L1_distance, cosine_distance
+from .model import create_init, Model
+from .optimizers import Adam, RAdam, SGD, Optimizer, ADAM_DEFAULTS, SGD_DEFAULTS
+from .schedules import (
```

Comment:
Maybe a personal preference but I typically use 2 lines for imports if they break (just takes up too much space otherwise and code becomes hard to read). I can fix this in this PR if you want.